database/repositories/helper.py

```diff
@@ -6,10 +6,11 @@
 
 
 class HelperRepository(Repository):
-    def __init__(self, sessionmaker: session, user_rank_model: Base, helper_range_model: Base):
+    def __init__(self, sessionmaker: session, user_rank_model: Base, helper_range_model: Base, helper_reward_model: Base):
         super(HelperRepository, self).__init__(sessionmaker=sessionmaker, model=user_rank_model)
-        self.user_model = user_rank_model
-        self.helper_model = helper_range_model
+        self.user_rank_model = user_rank_model
+        self.helper_range_model = helper_range_model
+        self.helper_reward_model = helper_reward_model
```

Comment:
w sumie można się pozbyć przedrostka `helper_` w tych dwóch polach, bo i tak wiadomo że to od helpera